appservice/src/disconnectRepo.ts

```diff
@@ -5,17 +5,16 @@
 
 import { SiteSourceControl } from 'azure-arm-website/lib/models';
 import { MessageItem } from 'vscode';
-import { AzureTreeItem, IActionContext } from 'vscode-azureextensionui';
-import { editScmType, ISiteTreeRoot } from './';
+import { IActionContext, ISubscriptionContext } from 'vscode-azureextensionui';
+import { editScmType, SiteClient } from './';
 import { ext } from './extensionVariables';
 import { localize } from './localize';
 import { ScmType } from './ScmType';
 
-export async function disconnectRepo(context: IActionContext, node: AzureTreeItem<ISiteTreeRoot>): Promise<void> {
-    const sourceControl: SiteSourceControl = await node.root.client.getSourceControl();
+export async function disconnectRepo(context: IActionContext, client: SiteClient, subscriptionContext: ISubscriptionContext): Promise<void> {
```

Comment:
nit: If you are also bothered by this, feel free to fix it, but if not, don't bother.

...but I hate how the order of the parameters is different between this and `editScmType` 😂

Reply:
Lol Nathan I agree haha. Also we usually put `IActionContext` first - so keep that in mind assuming you make them consistent